src/client/components/ReviewEditorComponent/ReviewEditorComponent.js

```diff
@@ -0,0 +1,64 @@
+import React ,{useState}  from 'react';
+import PropTypes from 'prop-types';
+import Button from '../Button/Button';
+import unknownUser from '../../assets/images/portraits/unknown-black.jpg';
+import './ReviewEditorComponent.style.css';
+
+export default function ReviewEditorComponent({
+  avatarUrl,
+  fullName,
+  inputText,
+  placeHolderText,
+  buttonStyle,
+}) {
+  const [inputValue,setInputValue] = useState()
+  const onChange = (event) => {
+    setInputValue((event.target.value))
+  };
+  return (
+    <div className="container">
+      <div className="top-part">
+        <div className="profile-picture">
+          <div className="circle-mask">
+            <img
+              src={avatarUrl || unknownUser}
+              alt="Portrait of user"
+              className="user-photo"
+            />
+          </div>
+        </div>
+        <div className="user-info">
+          <div className="user-name">{fullName}</div>
+        </div>
+      </div>
+      <div className="input-part">
+        <textarea value={inputValue} placeholder={placeHolderText} onChange={onChange}/>
+      </div>
+      <div className="button-part">
+        <Button
+          buttonName="Send review"
+          style={buttonStyle}
+          type="submit"
+          onClick={() => {
+            ('clicked');
+          }}
```

Comment:
@pdsorensen Since the task/issue hasn't been cleared here I need the suggestion to make this onClick functional.

Reply:
I don't believe the backend is ready? Maybe ask in slack and ask whether the backend for adding reviews has been implemented. 

If not, just leave the function with an empty body.